ssclj/jar/src/main/clojure/com/sixsq/slipstream/ssclj/db/database_binding.clj

```diff
@@ -52,56 +52,62 @@
       :data
       deserialize))
 
-(defn dispatch-fn   
+(defn dispatch-fn
   [collection-id options]
   collection-id)
 
 (defmulti  find-resources dispatch-fn)
 (defmethod find-resources :default
-  [collection-id options]  
-  (->>  (select resources (where {:id [like (str collection-id"%")]}))
-        (map :data)
-        (map deserialize)))
-
-(defn- delete-resource 
+  [collection-id options]
+  (let [{:keys [offset limit]} (u/offset-limit options)]
+    (if (neg? limit)
+      []
+      (->> (select resources
+                   (where {:id [like (str collection-id"%")]})
+                   (kc/limit offset)
+                   (kc/offset limit))
```

Comment:
@schaubl @st 
This [two lines](https://github.com/slipstream/SlipStreamServer/blob/issue_%23324-ssclj_pagination/ssclj/jar/src/main/clojure/com/sixsq/slipstream/ssclj/db/database_binding.clj#L67-L68) look suspicious.
We had a look with @schaubl without understanding why it works anyway.
I won't merge it until it's fixed or understood.
Cheers.